database/postgres_store.go

```diff
@@ -30,19 +34,81 @@ func (d *postgresStore) Close() {
 	d.DB.Close()
 }
 
-func (d *postgresStore) SaveRequestEntry(entry RequestEntry) error {
-	query := `INSERT INTO rpc_endpoint_requests
-	(id, received_at, request_duration_ms, is_batch_request, num_request_in_batch, http_method, http_url, http_query_param, http_response_status, ip_hash, origin, host, error) VALUES (:id, :received_at, :request_duration_ms, :is_batch_request, :num_request_in_batch, :http_method, :http_url, :http_query_param, :http_response_status, :ip_hash, :origin, :host, :error)`
+func (d *postgresStore) SaveRequestEntries(entries []RequestEntry) error {
+	tx, err := d.DB.Beginx()
+	var (
+		valStrings []string
+		valArgs    []interface{}
+	)
+	count := reflect.TypeOf(RequestEntry{}).NumField()
+	for i, entry := range entries {
+		valStrings = append(valStrings, fmt.Sprintf("($%d,$%d,$%d,$%d,$%d,$%d,$%d,$%d,$%d,$%d,$%d,$%d,$%d,$%d)",
+			i*count+1, i*count+2, i*count+3, i*count+4, i*count+5, i*count+6, i*count+7, i*count+8, i*count+9, i*count+10, i*count+11, i*count+12, i*count+13, i*count+14))
```

Comment:
Not finding right way to handle this.

Reply:
You can just call `NamedExecContext` with a slice of structs
```
query = "insert into rpc_endpoint_requests (id, received_at, inserted_at, request_duration_ms, ...) values (:id, :received_at, ...)
d.DB.NamedExecContext(ctx, query, entries)
```

Doesn't it work as expected?